scripts/mods/player_list_show_loadout/player_list_show_loadout.lua

```diff
@@ -366,6 +366,11 @@ mod.create_loadout_display = function(self, player_list_scenegraph)
 	}
 end
 
+-- This mod doesn't work properly in the Chaos Wastes (known as "deus" mode in the game source).
+local function is_in_chaos_wastes()
+	return (Managers.state.game_mode:game_mode_key() == "deus")
```

Comment:
Please check the mechanism instead of the gamemode:

```suggestion
	return (Managers.mechanism:current_mechanism_name() == "deus")
```